onnxruntime/core/providers/cpu/rnn/deep_cpu_gru.cc

```diff
@@ -313,7 +297,7 @@ Status DeepCpuGruOp::ComputeImpl(OpKernelContext& context) const {
 
   // GRU outputs are optional but must be in the same order
   TensorShape Y_dims{seq_length, num_directions_, batch_size, hidden_size_};
-  Tensor* Y = context.Output(/*index*/ 0, Y_dims);  // TODO: Adjust for however optional outputs gets implemented
+  Tensor* Y = context.Output(/*index*/ 0, Y_dims);  // TODO: Adjust for how optional outputs gets implemented
```

Comment:
>// TODO: Adjust for how optional outputs gets implemented [](start = 52, length = 57)

This has already been adjusted (optional inputs return nullptr) so this TODO can be removed.